build/wpt_test.bzl

```diff
@@ -133,14 +140,20 @@ def _wpt_wd_test_gen_impl(ctx):
     file, WPT test scripts, associated JSON resources.
     """
     src = ctx.actions.declare_file("{}.wd-test".format(ctx.attr.test_name))
+
+    # Both the .js and the .js.map file are included, and we need to tell them apart.
```

Comment:
How come? Can you add a link to WPT documentation or an example?

Reply:
That's actually down to Bazel ts_project. I'll reword for clarity.

Reply:
Actually I'm trying another trick that might work better on Windows. We'll see.